### ISSUE TITLE
Data protection incorrect documentation for storing keys to Azure Blob storage

### ISSUE THREAD

**Issue by janissimsons** [dotnet/AspNetCore.Docs#26306]
This code snippet is wrong, because method 'PersistKeysToAzureBlobStorage' does not take BlobContainerClient as a parameter.
It accepts connection string, container name and blob name directly, or it accepts BlobClient with a specific name. But it does not accept BlobContainerClient. Please fix it.

```
string connectionString = "<connection_string>";
string containerName = "my-key-container";
BlobContainerClient container = new BlobContainerClient(connectionString, containerName);

// optional - provision the container automatically
await container.CreateIfNotExistsAsync();

services.AddDataProtection()
    .PersistKeysToAzureBlobStorage(container, "keys.xml");
```

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 59850a13-1322-0c54-9fd3-75ce837c2b1a
* Version Independent ID: 7d481dae-d27d-909c-a35d-bad66c7508d2
* Content: [Key storage providers in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/security/data-protection/implementation/key-storage-providers?view=aspnetcore-6.0&tabs=visual-studio)
* Content Source: [aspnetcore/security/data-protection/implementation/key-storage-providers.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/security/data-protection/implementation/key-storage-providers.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-security**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by guardrex** [comment]
Thanks for the report, @janissimsons.

For reference, the update was made by @v-xuto on ...

**Update key-storage-providers.md**
https://github.com/dotnet/AspNetCore.Docs/pull/20397

Here's the specific line change on that PR ...

https://github.com/dotnet/AspNetCore.Docs/pull/20397/files#diff-17985bf3ee11cbca689fb3e5100ed2e889d4497676c4467ae372ca2d76652332R50

**Comment by v-xuto** [comment]
@janissimsons, @guardrex We have created a PR (https://github.com/dotnet/AspNetCore.Docs/pull/26360) to fix this issue. Please review this PR.

**Comment by Rick-Anderson** [comment]
Fixed in #26360
